views/partials/tips.hbs

```diff
@@ -1,4 +1,175 @@
+
 {{!< default }}
+
+<div id="password-advice" class="section-container">
+  <div class="section-wrapper">
+  <h3>Passwords</h3>
+  <h5>The Key to Account Security</h5>
+  <p>Your passwords protect all of the personal information that resides in your online accounts, from bank balances and credit card numbers to your home address and private messages. Passwords are so valuable that thousands of passwords are stolen every day and accounts are traded on the black market.  Take these six steps to protect yours.</p>
+  </div>
+</div>
+<div id="tips-container" class="section-container">
+    <div class="tips-wrapper">
+      <div class="tip-container">
+          <div class="tip">
+              <div class="svg-background">
+                <img src="img/Password_Tip_1@2x.png" alt="Account Security Tip #1">
+                <span class="tip-numeral medium">1</span>
+              </div>
+              <div class="tip-header">
+                  <h3>Use a different password<br>for every account</h3>
+              </div>
+              <p>You can't prevent a data breach, but you can limit your exposure by always using different passwords for different websites.</p>
+          </div><!--closes tip-->
+      </div><!--closes tip-container 1-->
```

Comment:
This is a lot of duplicated code, but I'm not sure if somehow converting the tips into a JSON object and trying to do loops and templates makes it worse and more convoluted.

```js
{
  "tips": [
    {
      "numeral": 1,
      "image_src": "img/Password_Tip_1@2x.png",
      "image_alt": "Use a different password for every account",
      "header": "<h3>Use a different password<br/>for every account</h3>",
      "description": "<p>You can't prevent a data breach, but you can limit your exposure by always using different passwords for different websites.</p>"
    },
   {"numeral": 2, ...},
   {"numeral": 3, ...},
   {"numeral": 4, ...},
   {"numeral": 5, ...},
   {"numeral": 6, ...}
  ]
}
```

We probably don't even need `"numeral"` and can just use the array index. And since the `"image_src"` has the index in the filename, we can probably lose that field too (as long as the HBS loop has access to it's index).

Even the `"image_alt"` could probably be generated by stripping out any HTML tags from the `"header"` field (using DOMPurify or a RegEx or whatever)...

Reply:
I also considered this and I'm not sure which is the better option! It would definitely be cleaner and less 'repeated code stenchy' to bring them in as a JSON object. I will have to explore the looping in Handlebars - I'm brand new to .hbs and am still finding my way around! I can't imagine that you can't access the index (but you never know).

Reply:
There is an HBS looping example at https://github.com/mozilla/blurts-server/blob/d963172fdc5fe3b5dc5d2a02977e5c109715d573/views/scan.hbs#L58-L74